moped-database/metadata/tables.yaml

```diff
@@ -5216,55 +5216,58 @@
     - role: moped-admin
       permission:
         columns:
-          - is_deleted
-          - original_fk
-          - table
+          - attributes
           - component_archtype_id
           - component_id
-          - feature_id
-          - project_id
-          - attributes
           - component_name
-          - project_name
-          - geography
           - council_districts
+          - feature_id
+          - geography
+          - is_deleted
           - length_feet
+          - line_representation
```

Comment:
The `line_representation` column is the only change in the metadata. It is used to divide line and point features when rendering and styling them in the map.